Sources/TSCBasic/FileSystem.swift

```diff
@@ -347,9 +347,9 @@ private struct LocalFileSystem: FileSystem {
         return FileInfo(attrs)
     }
 
-    func hasQuarantineAttribute(_ path: AbsolutePath) -> Bool {
+    func hasAttribute(name: String, _ path: AbsolutePath) -> Bool {
 #if canImport(Darwin)
```

Comment:
Doesn't look like `getxattr` is available with `import Glibc` so I'm not adding a check for Linux in this condition for now.

Reply:
Ah, okay; that seems like a different bug to resolve separately.